sklearn/ensemble/tests/test_bagging.py

```diff
@@ -401,28 +400,28 @@ def test_error():
     base = DecisionTreeClassifier()
 
     # Test max_samples
-    assert_raises(ValueError,
-                  BaggingClassifier(base, max_samples=-1).fit, X, y)
-    assert_raises(ValueError,
-                  BaggingClassifier(base, max_samples=0.0).fit, X, y)
-    assert_raises(ValueError,
-                  BaggingClassifier(base, max_samples=2.0).fit, X, y)
-    assert_raises(ValueError,
-                  BaggingClassifier(base, max_samples=1000).fit, X, y)
-    assert_raises(ValueError,
-                  BaggingClassifier(base, max_samples="foobar").fit, X, y)
+    with pytest.raises(ValueError):
```

Comment:
these checks could be parametrized (not super important though, you don't have to)